repositories.yaml

```diff
@@ -496,3 +496,25 @@ applications:
     channels:
       - v1_name: firefox-klar-ios
         app_id: org.mozilla.ios.Klar
+
+  - app_name: bergamot
+    canonical_app_name: Bergamot Translator
+    app_description: Web extension for on-device machine translation in Firefox
+    url: https://github.com/mozilla-extensions/bergamot-browser-extension
+    notification_emails:
+      - epavlov@mozilla.com
+    branch: main
+    metrics_files:
+      - extension/ts/background-scripts/background.js/telemetry/metrics.yaml
+    ping_files:
+      - extension/ts/background-scripts/background.js/telemetry/pings.yaml
+    dependencies:
+      - glean-core
+    retention_days: 180
+    channels:
+      - v1_name: bergamot
```

Comment:
We should add descriptions for what these variants are.

Reply:
bergamot - the extension is Implemented as an infobar, integrated into Firefox

bergamot-cross-browser - the extension is Implemented as a popup, can be installed in Firefox or Chrome